detox/setupJest.js

```diff
@@ -0,0 +1,26 @@
+const yargs = require('yargs');
```

Comment:
Does this file contains only test helper functions? if it does, let's move it to test folder.

Reply:
The `detox/test` path won't work as it's a separate project with a package.json and the module resolution won't get for example yargs as a package. I could move it in a `detox/__tests__` folder if you like, that should work

Reply:
yes, that's what I meant, `detox/test` is an e2e project

Reply:
👍